ush/hafs_ww3_restart_interp.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@Author: Ali Salimi-Tarazouj
+
+Regrid WW3 restart NetCDF file using ESMPy:
+https://earthsystemmodeling.org/esmpy/
+
+Usage:
+    mpirun -n 1 python hafs_ww3_restart_interp.py \
+        --restart_file <restart.nc> \
+        --src_scrip <src_scrip.nc> \
+        --dst_scrip <dst_scrip.nc> \
+        --mask_file <ww3_grid_a.msk> \
+        --output_file <output.nc>
+"""
```

Comment:
@AliS-Noaa For the documentation header, could you please help to update this section following an example (https://github.com/hafs-community/HAFS/blob/85d914a50ab6f7aacfc2bbc0e99c172d55095d00/ush/hafs_mom6_gfs_forcings.py#L1C1-L11C81). Thanks!